src/js/view/view.js

```diff
@@ -249,11 +249,21 @@ define([
             }
 
             _container = _playerElement.getElementsByClassName('jw-main')[0];
```

Comment:
Seems like this is causing some issues with jwplayer-commercial/test/vast-linear.html due to it looking for the jw-main class